src/main/java/com/jcabi/http/mock/MkAnswer.java

```diff
@@ -52,6 +52,7 @@
  * @since 0.10
  */
 @Immutable
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
@westonized I think it's time to add a puzzle to split this class somehow, as it gets too big